### PR TITLE
chore: export reused api explorer functionality from root index.ts

### PR DESCRIPTION
**Pull request by josephaxisa**

### CODE REVIEW

**Review by jkaster — COMMENTED** [review]
Looking good so far. I'll hold final approval for when it's out of draft mode.

- Running `yarn test:apix` I get a warning about an improper teardown, but `yarn test:apix --detectOpenHandles` doesn't show any warnings or details at all
- Running `yarn dev:apix` I get:

```
Uncaught TypeError: Failed to construct 'URL': Invalid base URL
at App.tsx:43
```

**Comment by josephaxisa** [comment]
@jkaster `yarn dev:apix` works fine on my end, but `yarn dev:xapix` does not.  Seems like the test issue has been there a while and is unrelated to these changes.

File: packages/api-explorer/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+export * from './ApiExplorer'
+export { Loader } from './components'
+export { store } from './state'
```
[the remark this answers is not in the document]
Comment (josephaxisa) [reply packages/api-explorer/src/index.ts:28]: I'll switch to named exports from 'ApiExplorer' for consistency

File: packages/api-explorer/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+export { ApiExplorer } from './ApiExplorer'
```
Comment (bryans99) [inline packages/api-explorer/src/index.ts:26]: Any compile issues with this? We've had to use export * elsewhere. Are there any other exports? if no consider switching to export *

**Review by bryans99 — DISMISSED** [review]
LGTM aside from export {} which we've had issues with.

I have not tested though and wont have a chance to

**Review by jkaster — APPROVED** [review]
LGTM now. Tested both extension and browser environment

**Pull request merged**